realm/realm-library/src/main/cpp/util.cpp

```diff
@@ -195,11 +195,14 @@ void ThrowException(JNIEnv* env, ExceptionKind exception, const std::string& cla
     env->DeleteLocalRef(jExceptionClass);
 }
 
-void ThrowRealmFileException(JNIEnv* env, const std::string& message, realm::RealmFileException::Kind kind)
+void ThrowRealmFileException(JNIEnv* env, const std::string& message, realm::RealmFileException::Kind kind, const std::string& path)
 {
-    jclass cls = env->FindClass("io/realm/exceptions/RealmFileException");
+    jclass jrealm_file_exception_cls = env->FindClass("io/realm/exceptions/RealmFileException");
```

Comment:
This is not called often, just when throwing exception ... do you still think it's worth caching?

Reply:
yes, please, then mistakes like https://github.com/realm/realm-java/pull/5297/files#diff-782ab33a7d2f32511e3d7879a2fabe3eR241 won't happen.

plus it could save a few lines of code.